deployment/sql/checkpoints.sql

```diff
@@ -0,0 +1,6 @@
+CREATE TABLE checkpoints (
```

Comment:
https://github.com/azavea/osmesa/blob/master/src/analytics/Makefile should also be moved into `deployment/sql` and an entry added for `checkpoints` in the `db/all` target.